build.sbt

```diff
@@ -30,5 +41,4 @@ lazy val settings = Seq(
     val required = "1.8"
     val current  = sys.props("java.specification.version")
     assert(current == required, s"Unsupported JDK: java.specification.version $current != $required")
-  }
-)
+  }) ++ formattingSettings
```

Comment:
I don't mind not having the `formattingSettings` applied to the root project. Regarding the `build.sbt` and `/project` files, I don't think the extra boilerplate justifies it, as they're not modified that often, but I don't mind adding it if you think it's worth it.